services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -223,6 +223,11 @@ class DynamicSidecarSettings(BaseCustomSettings):
         description="Names the traefik zone for services that must be accessible from platform http entrypoint",
     )
 
+    DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS: dict[str, str] = Field(
+        ...,
+        description="provided by ops, are injected as service labels when starting the dy-sidecar",
```

Comment:
suggestion: explain what it is. who defines them is pretty clear

Reply:
let's see if now it looks better